fixtures/chauvet-professional/vesuvio-rgba.json

```diff
@@ -0,0 +1,699 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Vesuvio RGBA",
+  "shortName": "Chauvet Jet",
+  "categories": ["Smoke", "Color Changer"],
+  "meta": {
+    "authors": ["Beerady", "Luke Nelson"],
+    "createDate": "2021-09-09",
+    "lastModifyDate": "2022-10-17"
```

Comment:
Please update both `createDate` and `lastModifyDate` for new fixtures; `createDate` is rather the first publish date.

Reply:
Ahh, ok, thanks for letting me know :)